CONTRIBUTING.md

```diff
@@ -51,6 +51,18 @@ If you'd like to contribute a post to the layer5.io/blog, please open an issue a
 
 The following list of instructions pertain to commonplace site updates by contributors.
 
+## Things to keep in mind
+
+  1. Use only the theme colors available at [themeStyles.js](https://github.com/layer5io/layer5/blob/master/src/theme/app/themeStyles.js) instead of external color codes.
+  2. Avoid adding `rel="noreferrer"` attribute to internal links on the site.
```

Comment:
Reinforced by an eslint pre-commit hook, too.

Reply:
You mean this needs to be configured, right?
Or, we are good with this PR and configure it later?